editor/import/3d/resource_importer_scene.cpp

```diff
@@ -1298,8 +1298,15 @@ Node *ResourceImporterScene::_post_fix_animations(Node *p_node, Node *p_root, co
 					//fill with default values
 					List<ImportOption> iopts;
 					get_internal_import_options(INTERNAL_IMPORT_CATEGORY_ANIMATION, &iopts);
+					bool has_slices = true;
+					String slice_str = "slice_" + itos((int)anim_settings["slices/amount"] + 1);
```

Comment:
I don't think it should be + 1, since my understanding is slices are zero indexed.

For example, if I have slices/amount = 2, I want `slice_0/something` and `slice_1/something`, but by the time we hit `slice_2/something` , we should set has_slices to false. Hence it should not be adding 1 to the slices/amount.

Reply:
Slices are 1-indexed inside the `.import` settings file.
Everything is working normally. 

Here are the `.import` files generated by the original Godot build vs this PR if you want to check:

[ori vs fixed import files.zip](https://github.com/user-attachments/files/17972242/ori.vs.fixed.import.files.zip)

Reply:
OMG. Never, anywhere else in the engine, have I seen one-indexed arrays used for anything. utterly crazy.

Okay I did a grep for `itos` and `+ 1` and there are only a couple other places: `dock_split_` in the editor dock settings is one-indexed. There's some dialog box code for connection bindings that randomly does this. And finally the list of draw passes for a GPUParticles3D adds one I think just for the header visibility in the editor.

Anyway I understand now, the slices are designed to be one-indexed and different from the rest of godot.